_posts/2019-08-19-this-week-in-rustfest-barcelona-scholarships.md

```diff
@@ -0,0 +1,38 @@
+
+Many of this year's improvements are centered around making the application process itself more inclusive, as part of this effort we are proud to announce our Scholarships are available in 3 languages: english, [spanish](https://barcelona.rustfest.eu/scholarships/es/) and [catalan](https://barcelona.rustfest.eu/scholarships/ca/)! Read all the other details on the [Scholarships page](https://barcelona.rustfest.eu/scholarships/) and send us feedback, we'd love to hear your feedback and ideas how to further improve our process!
+
+
+## A final round of Community Tickets available later today
+
+Another conscious, long-standing effort of the RustFest organizing team is ensuring we keep the conference accessible even for paying attendees. We have historically made sure [we kept ticket prices affordable](https://barcelona.rustfest.eu/tickets/), while also ensuring that the conference stays financially stable.
+
+Our Community Tickets are the most affordable general-availability ticket type available for everyone who wants to join us at RustFest, but to support the above-mentioned financial stability, we need to pace their availability based on our budget situation. We are happy to announce, that we have managed to secure enough funds to be **[releasing one last batch of Community Tickets](https://ti.to/rustfest/barcelona2019/) at 4 PM UTC (18:00 CEST) later today!**
+
+Please note, this is the final round of Community Tickets, there won't be any more rounds and no waitlist will be made available after this round is sold out. Our full price tickets will remain on sale until we reach venue capacity. Just a small reminder, if you can afford them, please opt for the Full Price tickets, supporting us, other ticket holders and our above-mentioned Scholarship (as 10% of all Full Price tickets goes straight into our Scholarship fund).
+
+
+## Extending the CFP deadline
+
+We have alredy received a ton of interesting submissions to our Call for Proposals (thank you!), but due to the raging summer and vacation season we have decided to give you all a bit more time to cook up a proposal: we are extending the CFP with an extra week!
+
+[Head on to our CFP page to submit a proposal](http://cfp.rustfest.eu/events/rustfest-barcelona-2019), and please [help us spread the word](https://twitter.com/RustFest/status/1153566837386481664), to ensure our [CFP commitee](https://barcelona.rustfest.eu/cfp-committee/) has a real tough time deciding which excellent talk to showcase on the RustFest Barcelona stage this November!
```

Comment:
the cfp committee site is currently empty, not sure if we should link to it.

Reply:
SO YOU SHOULD FIX IT

(yeah I realized that, we should get that populated, but no harm in linking to it IMHO)